cmd/analyze/analyze.go

```diff
@@ -88,6 +122,8 @@ func init() {
 	AnalyzeCmd.Flags().BoolVarP(&nocache, "no-cache", "c", false, "Do not use cached data")
 	// anonymize flag
 	AnalyzeCmd.Flags().BoolVarP(&anonymize, "anonymize", "a", false, "Anonymize data before sending it to the AI backend. This flag masks sensitive data, such as Kubernetes object names and labels, by replacing it with a key. However, please note that this flag does not currently apply to events.")
+	// alert flag
+	AnalyzeCmd.Flags().StringVarP(&alert, "alert", "", "", "Alert to be analyzed")
```

Comment:
Let's offer a shorthand flag also, it doesn't need to be "-a"

Reply:
What do you think about "l" ?

Reply:
that's fine, might be easier to make it a less ambiguous char like `g` or `y`